_g2/tests/test-widget-new-posts.php

```
@@ -0,0 +1,105 @@
+
+	/**
+	 * 空の $instance で form() を呼び出しても E_WARNING が発生しないことを確認する
+	 *
+	 * ウィジェット初回追加時（保存前）は $instance が空配列で渡されるため、
+	 * wp_parse_args() によるデフォルト値補完が正しく機能していることを検証する。
+	 */
+	public function test_form_with_empty_instance_does_not_trigger_warning() {
+		// E_WARNING が発生した場合にテストを失敗させるためのエラーハンドラーを登録する
+		$warning_occurred = false;
+		set_error_handler(
+			function( $errno, $errstr ) use ( &$warning_occurred ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$errstr'. (undefined)

(UnusedFormalParameter)